collections/machine-learning/index.md

```diff
@@ -1,5 +1,6 @@
 ---
 items:
+ - AMAI-GmbH/AI-Expert-Roadmap
```

Comment:
Can yo please move this to the bottom of the list? Thanks!

Reply:
Ok, I moved it to the bottom. Thanks!